routes/index.js

```diff
@@ -5,6 +5,7 @@ app.use('/healthcheck', require('./healthCheck.js'))
 app.use('/auth', require('./auth.js'))
 app.use('/users', require('./users.js'))
 app.use('/members', require('./members.js'))
+app.use(require('./tasks'))
```

Comment:
I understand what you're doing here, but I'm not very sure about it 😅 
@ankurnarkhede @whyDontI What do you suggest?

Reply:
I will make it easy for you ;)